modules/kimberliteBidAdapter.js

```diff
@@ -34,19 +34,40 @@ const converter = ortbConverter({
   imp (buildImp, bidRequest, context) {
     const imp = buildImp(bidRequest, context);
     imp.tagid = bidRequest.params.placementId;
+    const mediaTypes = bidRequest.mediaTypes;
+    if (imp[VIDEO]) {
+      deepSetValue(imp, 'video.linearity', mediaTypes[VIDEO].linearity || 1);
+      deepSetValue(imp, 'video.protocols', mediaTypes[VIDEO].protocols || [3, 6]);
+      deepSetValue(imp, 'video.placement', mediaTypes[VIDEO].placement || 1);
```

Comment:
I think the ortb convertor gives you all these for free, no need to copy them. Is that correct @dgirardi ?

Reply:
I think this is to set default values, which the library does not provide.

Reply:
Ah I see @os-solta these defaults aren't good guesses, you're better off requiring the publisher make a choice

Reply:
Looks like i'll just skip these defaults here, thanks.